src/assets/scripts/client/navigationLibrary/NavigationLibrary.js

```diff
@@ -45,6 +45,33 @@ export default class NavigationLibrary {
     }
 
     /**
+     *
+     * @property drawSids
+     * @return
+     */
+    get drawSids() {
+        return this._sidCollection.draw;
+    }
+
+    /**
+     *
+     * @property sidCollection
+     * @return
+     */
+    get sidCollection() {
+        return this._sidCollection;
+    }
```

Comment:
Are we unable to mutate `instance.sidCollection` because it's a getter? Or does this not actually protect the contents of `instance._sidCollection`?

(purely educational question only)